github/teams.go

```diff
@@ -494,77 +686,46 @@ func (s *TeamsService) RemoveTeamProject(ctx context.Context, teamID int64, proj
 	return s.client.Do(ctx, req, nil)
 }
 
-// IDPGroupList represents a list of external identity provider (IDP) groups.
```

Comment:
Why are lines 497 to the end of the file being deleted in this PR?
Oh, it looks like you are also trying to reorganize the files in this PR as well?
Sorry, but please handle file reorganization in a separate PR.

Attempting to address the changes listed in the announcement: https://developer.github.com/changes/2020-01-21-moving-the-team-api-endpoints/
while at the same time reorganizing files and moving code around makes this PR much much larger than it needs to be and is not reviewable as such.

So please keep this PR focused just on the checklist I made in #1387 and revert the rest of the changes and make a separate PR for those.  Thanks.

Reply:
Sure thing.  Must've missed the list.  Working on reverting the changes now.  Thanks!